lakey_service/downloader/executors/athena.py

```diff
@@ -7,6 +7,7 @@
 import requests
 import pandas as pd
 
+from .baseexecutor import BaseExecutor
```

Comment:
if we're in folder `executors` than because of the context you can easily just rename `baseexecutor` to `base`. Since sticking to the `baseexecutor` would suggest that we should for the consistency reasons rename `athena` to `athenaexecutor` and similar for `databricks`.